app/gui2/src/components/GraphEditor/widgets/WidgetFunction.vue

```
@@ -141,6 +142,16 @@ const visualizationConfig = computed<Opt<NodeVisualizationConfiguration>>(() =>
 const visualizationData = project.useVisualizationData(visualizationConfig)
 const widgetConfiguration = computed(() => {
   if (props.input.dynamicConfig?.kind === 'FunctionCall') return props.input.dynamicConfig
+  if (props.input.dynamicConfig?.kind === 'OneOfFunctionCalls' && methodCallInfo.value != null) {
+    const cfg = props.input.dynamicConfig
+    const info = methodCallInfo.value
+    const name = entryQn(info?.suggestion)
+    const usedFunctionConfig = cfg.possibleFunctions.get(name)
+    if (!usedFunctionConfig) {
+      console.error(`Expecting one of ${cfg.possibleFunctions}, found ${name}.`)
```

Is this really an error? Dynamic configs are not really required to be exhaustive.

---

The idea was to catch such situations, rather than silently ignoring them. But I guess you’re right.